resource-definitions/hpa-driver/horizontalpodautoscaler/README.md

```diff
@@ -0,0 +1,3 @@
+This section contains a Resource Definition example for handling Kubernetes [`HorizontalPodAutoscaler`](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/) by using the [`hpa`](https://developer.humanitec.com/integration-and-extensions/drivers/k8-drivers/horizontal-pod-autoscaler/) Driver. If you have special requirements for your `HorizontalPodAutoscaler` implementation, you can [see this other example using the `template` Driver](https://developer.humanitec.com/examples/resource-definitions/template-driver/horizontalpodautoscaler/).
+
+You can find a Score file example using the `horizontal-pod-autoscaler` resource type [here](https://developer.humanitec.com/examples/score/horizontalpodautoscaler/).

```

Comment:
That Score file defines `params` for the `horizontal-pod-autoscaler` resource, but they are not honored by the Resource Definition and the values coded into the `driver_inputs` are used instead.

So we can either provide a second Score file without `params`, or add a comment into that file saying that the Resource Definition needs to support them. I suggest the latter.

```suggestion
You can find a Score file example using the `horizontal-pod-autoscaler` Resource Type [here](https://developer.humanitec.com/examples/score/horizontalpodautoscaler/). Note that the present Resource Definition will not honor the `params` set for the `horizontal-pod-autoscaler`, but instead use the values defined in its `driver_inputs`.
```

Reply:
Like commented above, `hpa` Driver by default to show the out-of-the-box approach first.

Reply:
Suggesting to keep it like this.